go.mod

```diff
@@ -27,3 +27,5 @@ require (
 	go.opencensus.io v0.21.0
 	golang.org/x/xerrors v0.0.0-20190513163551-3ee3066db522
 )
+
+go 1.13
```

Comment:
Intentional?

Reply:
Bah.

Reply:
I've dropped this to 1.12 so it stops complaining. Although... We're going to merge some TLS changes pretty soon that'll force 1.13.